activecampaign.php

```diff
@@ -59,7 +59,7 @@ function activecampaign_form_source($settings, $form, $static = false) {
 		if ($static) {
 			$source .= "static=1&";
 		}
-		$source .= sprintf("id=%d", $form["id"]);
+		$source .= sprintf("id=%d&%s", $form["id"], strtoupper(uniqid()));
```

Comment:
The problem I have is it has to be a unique URL for each form embed. So if we have the same form embedded twice, wouldn't it still be the same version or fingerprint? What I need is:

`embed.php?id=27&something`
`embed.php?id=27&something_else`

Always has to be unique no matter how many of those `embed.php` lines there are.